models/stereo_camera/meshes/hokuyo.dae

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The `hokuyo.dae` meshes get repeated in the `stereo_camera` and the `depth_camera`. Is there a way we can have a specific model for each of the cameras? Example - Realsense camera: https://github.com/intel/gazebo-realsense/tree/master/models/realsense_camera